pkg/workloads/cortex/serve/start_uvicorn.py

```diff
@@ -15,11 +15,22 @@
 import uvicorn
 import yaml
 import os
+import json
 
 if __name__ == "__main__":
     with open("/src/cortex/serve/log_config.yaml", "r") as f:
         log_config = yaml.load(f, yaml.FullLoader)
 
+    has_multiple_servers = os.getenv("CORTEX_MULTIPLE_TF_SERVERS", None)
```

Comment:
The `None` here isn't necessary, but if you added it for clarity, feel free to leave it

Reply:
Removed it in both places.